src/core/Akka/Util/Internal/AtomicState.cs

```diff
@@ -94,16 +97,18 @@ public async Task<T> CallThrough<T>(Func<Task<T>> task)
             return result;
```

Comment:
Actual fix, this does not enforce that the underlying task be stopped, it is up to the task itself to use the cancellation token and stop itself. Will need to be propagated manually to anything that uses circuit breaker.